src/hooks/useKnockoutExplorer.ts

```diff
@@ -433,7 +433,14 @@ async function updateContextForSampleData(explorer: Explorer): Promise<void> {
 
   const data: SampledataconnectionResponse = await response.json();
   const sampleDataConnectionInfo = parseResourceTokenConnectionString(data.connectionString);
-  updateUserContext({ sampleDataConnectionInfo });
+  updateUserContext({
+    sampleDataConnectionInfo,
```

Comment:
No need to overwrite the feature flag. Just add this condition when deciding whether to render the button or not.